backend/nokia-innovative-project/src/main/java/com/nokia/library/nokiainnovativeproject/services/EmailService.java

```diff
@@ -1,32 +1,35 @@
 package com.nokia.library.nokiainnovativeproject.services;
 
 import com.nokia.library.nokiainnovativeproject.DTOs.Email;
-import com.nokia.library.nokiainnovativeproject.utils.EmailRecipients;
 import lombok.RequiredArgsConstructor;
 import org.springframework.mail.MailException;
 import org.springframework.mail.SimpleMailMessage;
-import org.springframework.mail.javamail.JavaMailSender;
-import org.springframework.retry.annotation.Backoff;
-import org.springframework.retry.annotation.Retryable;
+import org.springframework.scheduling.annotation.Async;
 import org.springframework.stereotype.Service;
 
+import java.util.List;
+
 @Service
 @RequiredArgsConstructor
 public class EmailService {
 
-	private final JavaMailSender emailSender;
-
-	@Retryable(
-			value = MailException.class,
-			maxAttempts = 5,
-			backoff = @Backoff(delay = 1000 * 60 * 10))
-	public void sendSimpleMessage(Email email) {
+	private final SendEmailService sendEmailService;
 
-		SimpleMailMessage message = new SimpleMailMessage();
-		message.setTo(EmailRecipients.recipients.toArray(new String[0]));
-		message.setSubject(email.getSubject());
-		message.setText(email.getMessageContext());
+	@Async
+	public void sendSimpleMessage(Email email, List<String> recipients) {
 
-		emailSender.send(message);
+		if(recipients.size()  > 0) {
+			SimpleMailMessage message = new SimpleMailMessage();
+			message.setTo(recipients.toArray(new String[0]));
+			message.setSubject(email.getSubject());
+			message.setText(email.getMessageContext());
+			try {
+				sendEmailService.send(message);
+			}
+			catch(MailException e){
+				// attempts to send an email failed
+				// we skip the problem
+			}
+		}
```

Comment:
to reduce indentation please invert if.
`if(recip.isEmpty){return;}
... rest of the code
`